### ISSUE TITLE
Wrong link

### ISSUE THREAD

**Issue by stefanak-michal** [php/doc-en#1737]
From manual page: https://php.net/context.ssl

---
**security_level** description contains link which redirects to 404 page.

**Comment by cmb69** [comment]
Indeed. That should probably be https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_get_security_level.html.
